pos_loyalty/tests/test_pos_loyalty.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2004-2018 Odoo SA
+# Copyright 2018 Lambda IS DOOEL <https://www.lambda-is.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests
+
+
+class TestPOSLoyalty(odoo.tests.HttpCase):
+    def test_pos_loyalty(self):
+        env = self.env(user=self.env.ref('base.user_admin'))
+
+        main_pos_config = env.ref('point_of_sale.pos_config_main')
```

Comment:
Hi @elBati, thanks for your PR.
Maybe it's possible to unset iotbox option on this config
This should prevent the POS to look for the iotbox

Reply:
Tried with

```
diff --git a/pos_loyalty/tests/test_pos_loyalty.py b/pos_loyalty/tests/test_pos_loyalty.py
index 4901bec3..ce4671df 100644
--- a/pos_loyalty/tests/test_pos_loyalty.py
+++ b/pos_loyalty/tests/test_pos_loyalty.py
@@ -10,6 +10,15 @@ class TestPOSLoyalty(odoo.tests.HttpCase):
         env = self.env(user=self.env.ref('base.user_admin'))
 
         main_pos_config = env.ref('point_of_sale.pos_config_main')
+
+        # Disabile "use_proxy", see point_of_sale/static/src/js/models.js
+        main_pos_config.iface_payment_terminal = False
+        main_pos_config.iface_electronic_scale = False
+        main_pos_config.iface_print_via_proxy = False
+        main_pos_config.iface_scan_via_proxy = False
+        main_pos_config.iface_cashdrawer = False
+        main_pos_config.iface_customer_facing_display = False
+
         target_product = env.ref('point_of_sale.letter_tray')
         free_product = env.ref('point_of_sale.desk_organizer')
         customer = env.ref('base.res_partner_2')
```

but still getting

```
            <div class="loader">
                <div class="loader-feedback">
                    <h1 class="message">Connecting to the IoT Box</h1>
                    <div class="progressbar">
                        <div class="progress" width="50%" style="width: 23%;"></div>
                    </div>
                    <div class="button skip">
                        Skip
                    </div>
                </div>
            </div>
```

Reply:
Also found `pos_customer_display` module, trying with 6ee9bcb2ce4304fe1f972fce7d39d50d207d935d

Reply:
I think that merging https://github.com/OCA/pos/pull/763 could fix your tests.